pytorch/buildspec.yml

```diff
@@ -1,7 +1,7 @@
   account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
Change line 72 to use cu112 folder

Reply:
line 72 here is `cuda_version: &CUDA_VERSION cu110`, which folder are you referring to?